### PR TITLE
Update Megahit from 1.1.3 to 1.2.9 version

### PR DESCRIPTION
**Pull request by PlushZ**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/megahit/megahit_wrapper.xml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <?xml version='1.0' encoding='utf-8'?>
-<tool id="megahit" name="MEGAHIT" version="@VERSION@.5">
+<tool id="megahit" name="MEGAHIT" version="@VERSION.0@">
```
Comment (bgruening) [inline tools/megahit/megahit_wrapper.xml:2]: You don't need to change that. The macro is enough. You could adopt the wrapper to the latest IUC standards if you like: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html?#tool-versions

**Review by bgruening — COMMENTED** [review]
Thanks! That looks great. I just have one small comment.

File: tools/megahit/megahit_wrapper.xml (hunk 1 above)
Comment (PlushZ) [reply tools/megahit/megahit_wrapper.xml]: Thank you, ok, I updated the wrapper according to iuc-standards.

Comment (bgruening) [reply tools/megahit/megahit_wrapper.xml]: Very cool, thanks a lot!

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks @PlushZ!

**Comment by bebatut** [comment]
Good job @PlushZ. Thanks a lot!

**Pull request merged**